### DISCUSSION TITLE
Extend theme with keyframes and animation

### DISCUSSION THREAD

**Discussion by go4cas** [unocss/unocss#2012] [Q&A] [answered]
I've been trying to implement a wiggle animation (sample from the Tailwind docs), but no luck:

[Playground](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwHIAuBTAHpgWhmwDscAnKAWwICZ0A%2BKKAKGeAAdZEUMB3OgAYoCEmCoIcBPmADmsuNnRQARrIzlsIAgDZBgxjiSZgAeg4MWZ8BAZA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOKrUwAFqhCoAXIhVqjAa1QBPSsJ2t9yo-bgB3eUTF6DDh7QAMAUgA0cACM3n60tnAwUMJYrOjQIPq0VDDCMKgKALQAzGhEAJS0ZP6Gnmq0AKxhEVExcQlJKWkZuagFRaQlZaqdpfa9ZTGgacAEtn0OzkSu7rRTM8GscKjCrKiZ2JkQ9PDY0sDptF2eA-0lpPlAA&options=N4IgzgLgTglgxhEAuAZgQwDZgKYBoTRoB2YKA9lALbLpbYC%2BQA)

Not sure what the correct approach should be?

**Comment by zyyv** [comment, marked as answer]
It has a bit complicated. [Playground](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwHIAuBTAHpgWhmwDscAnKAWwICZ0A%2BKKAKGeAAdZEUMB3OgAYoCEmCoIcBPmADmsuNnRQARrIzlsIAgDZBgxjiSZgAeg4MWZ8BAZA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHBgALVCFQAuRCtVxhWUMJjACe5YcMBrVAE9Kw7aysGbqgO7yiY3XAABtaeoQAMAKQANHAAjGGRiGpQxqzo0CB6VDBmqAoAtADMaEQAlGQeoYYArIlIMClYaRlZEDkwecWoZWRRlaGkgX0Dw57I9CnmBG76VXA%2BRH4BtLEcozak64bmINhEAGJN7nMLS3q0qMKsqPnY%2BRD0MLRbqpv9cMwPWDAzIaGn-nO2GkwA6z3eb1CAHoofNfIC6ADUHFWHBLtdblh7o84MDsKDUOCNqM3qRSpwgA&options=N4IgzgLgTglgxhEAuAZgQwDZgKYBoTRoB2YKA9lALbLpbYC%2BQA)

**Reply by go4cas** [reply]
Great! Thanks, @zyyv! This worked.

PS: When adding to the `vite.config.js` file, I had to remove the comma from this line: 50% { transform: rotate(3deg) },.

Thanks again!
